dashboard/modules/train/train_head.py

```diff
@@ -96,6 +102,31 @@ async def get_train_runs(self, req: Request) -> Response:
             content_type="application/json",
         )
 
+    async def _add_actor_status(self, train_runs):
+        actor_status_table = {}
+        try:
+            logger.info("Getting all actor info from GCS.")
+            request = gcs_service_pb2.GetAllActorInfoRequest()
+            reply = await self._gcs_actor_info_stub.GetAllActorInfo(request, timeout=5)
+            if reply.status.code == 0:
+                for message in reply.actor_table_data:
+                    actor_table_data = actor_table_data_to_dict(message)
+                    actor_status_table[actor_table_data["actorId"]] = actor_table_data[
+                        "state"
+                    ]
+        except Exception:
+            logger.exception("Error Getting all actor info from GCS.")
+
+        for train_run in train_runs.values():
+            train_run.controller_actor_status = actor_status_table.get(
+                train_run.controller_actor_id, None
+            )
```

Comment:
i thought we were removing controller_actor_status?

Reply:
Ah I see. I removed it.